### ISSUE TITLE
Allow locks to be stored outside of filesystem

### ISSUE THREAD

**Issue by gggeek** [kaliop/kaliop-lock-command-bundle#4]
In the scenario where one executes commands on multiple, separate servers, locking via filesystem can be not effective.

To allow locking commands across all servers, a shared-lock implementation is required.

Simfony provides this via the Lock component, which is available since version 3.4, so it makes a good candidate for replacing the LockHandler.

Moreover, LockHandler is deprecated and then removed in Sf 4

**Comment by bOnepain** [comment]
Yes indeed, I know this new component but haven't had time to use it in this bundle. I'll have a look at that as soon as possible.
